### ISSUE TITLE
Make failed on ARM64, undefined: DNSSnooper

### ISSUE THREAD

**Issue by openthings** [weaveworks/scope#2110]
2016-12-25, run "make"

**Platform Info:**
* Arch: ARM64
* Hardware: 96Boards
* CPU: Snapdragon 410C
* Host OS: Linaro based debian
* Docker container: aarch64/ubuntu:xenial 
* Toolchain:  golang1.7.4, node 6.9.2, npm 4.0.5
* Source: https://github.com/openthings/scope/tree/ARM64, branch ARM64

**Error info:** 
```
docker run --rm -ti \
	-v /media/linaro/zdata/src/github.com/openthings/scope:/go/src/github.com/weaveworks/scope \
	-v /media/linaro/zdata/src/github.com/openthings/scope/.pkg:/go/pkg \
	--net=host \
	-e GOARCH -e GOOS -e CIRCLECI -e CIRCLE_BUILD_NUM -e CIRCLE_NODE_TOTAL \
	-e CIRCLE_NODE_INDEX -e COVERDIR -e SLOW -e TESTDIRS \
	weaveworks/scope-backend-build SCOPE_VERSION=05869ab GO_BUILD_INSTALL_DEPS=-i prog/scope
make: Entering directory '/go/src/github.com/weaveworks/scope'
rm -f render/detailed/detailed.codecgen.go; unset GOOS GOARCH; env GOGC=off go build -i -ldflags "-extldflags \"-static\" -X main.version=05869ab -s -w" -tags 'netgo unsafe' ./render/detailed # workaround for https://github.com/ugorji/go/issues/145
# github.com/weaveworks/scope/probe/endpoint
probe/endpoint/reporter.go:37: undefined: DNSSnooper
Makefile:90: recipe for target 'render/detailed/detailed.codecgen.go' failed
make: *** [render/detailed/detailed.codecgen.go] Error 2
make: Leaving directory '/go/src/github.com/weaveworks/scope'
Makefile:67: recipe for target 'prog/scope' failed
make: *** [prog/scope] Error 2
```

**Comment by rade** [comment]
There is no official support for ARM in Scope, though PRs for that are certainly welcome..

**Comment by 2opremio** [comment]
Our CI makes sure arm 32 at least builds, but we don't test it. Unfortunately we don't even get to build arm64.

**Comment by arnulfojr** [comment]
I'm getting the same error, @openthings have you find a solution?

**Comment by arnulfojr** [comment]
I have managed to build it in arm64, maybe you could take a look into the PR? :) maybe someone else would as well like to use build it

**Comment by bboreham** [comment]
Closed by #3537
